include/swift/Serialization/ModuleFile.h

```diff
@@ -697,6 +697,9 @@ class ModuleFile : public LazyMemberLoader {
 
   virtual void loadAllMembers(Decl *D,
                               uint64_t contextData) override;
+private:
+  Decl* getOrFudgeOneMember(Decl *container, serialization::DeclID rawID);
```

Comment:
While 'fudge' is a perfectly cromulent verb, it's also not one we're currently using anywhere else, and I'd hesitate to add yet more terms since we already have several terms for the entities and actions involved in deserialization recovery: recover, missing-members, placeholders. Maybe 'getMemberOrMissingMemberPlaceholder', 'handleErrorAndRecover', and such?